### PR TITLE
Hotfix/1.53.2

### PR DESCRIPTION
**Pull request by tomlovesgithub**
---

## What's included?
Dependency updates, and work to deploy '_service offline_' placeholder.

## Who should test?
✅ Developers

## How to test?
See `package.json` and test service offline deployments.
Please only test against develop and staging environments.
Rollback via deployments on firebase console

## Risk - how likely is this to impact other areas?
🟠 Medium risk.

---

PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "deploy-develop": "vue-cli-service build --mode develop; firebase deploy --only hosting:develop-candidate-app -P develop",
     "deploy-staging": "vue-cli-service build --mode staging; firebase deploy --only hosting:staging-candidate-app -P staging",
     "deploy-production": "vue-cli-service build --mode production; firebase deploy --only hosting:production-candidate-app -P production",
+    "deploy-service-offline:develop": "rm -rf dist; mkdir dist; cp service-offline/index.html dist/index.html; firebase deploy --only hosting:develop-candidate-app -P develop",
```
Comment (HalcyonJAC) [inline package.json:17]: This deployment script doesn't work on Windows so I couldn't test the deployment

**Comment by HalcyonJAC** [comment]
This also has merge conflicts

**Review by HalcyonJAC — DISMISSED** [review]
It seems that using semicolon (;) to combine commands will not work on Windows. Could you create another script for Windows, please?

File: package.json (hunk 1 above)
Comment (andrew-isaac) [reply package.json:17]: Please action @HalcyonJAC  comments above :)

**Review by andrew-isaac — DISMISSED** [review]
Please action changes flagged by @HalcyonJAC

**Review by warrensearle — APPROVED** [review]
I think we're ok to proceed with the script only working for developers using mac/linux. After all that is what JAC in-house team use and I'd expect they'll be the only ones instructed to run the script.
That said @tomlovesgithub it _should_ be a case of simply replacing the `;` in your scripts with `&&` instead so we may as well make the windows lubbers happy too 😄 
This is a useful post: https://docgov.dev/posts/npm-scripts/

**Comment by HalcyonJAC** [comment]
I've run the script and it works well.

**Review by HalcyonJAC — APPROVED** [review]

**Pull request merged**